content/posts/unpublished/working-with-test-containers/post.md

```diff
@@ -0,0 +1,444 @@
+# Working with test containers
+
+
+Framework that simplifies the process of setting up, managing, and tearing down containerized environments for integration testing.
+
+
+![](assets/testcontainers.jpg)
+
+
+Authors: Yauheni Navosha
+Date: unpublished
+Category: backend
+
+
+tags: backend, tests, testing, integration testing, docker, containers
+
+---
+
+## Introduction
+
+
+We are going to create a Spring Boot project using Kafka, Spring Data JPA, MySQL database, Backbase [events](https://backbase.io/developers/documentation/backend-devkit/17.0.0/messaging/use-events/) for our scenario, where we implement a service which receives an event payload and persists the event data in the database.
```

Comment:
Please update introduction section what is test container and why we need it  then you can go ahead with when you need in order to enable test container

Reply:
Reworked, please have a look